### DISCUSSION TITLE
How to follow redirects with `Issuer.discover()`?

### DISCUSSION THREAD

**Discussion by viglucci** [panva/openid-client#524] [Q&A] [answered]
Hi!

I have used this package for a recent project and enjoy working with it. Thanks for all your hard work!

One issue I am having is that one configuration endpoint my application consumes performs a redirect. This results in an error: `OPError: expected 200 OK, got: 302 Found`.

Is there a way to have `Issuer.discover()` automatically follow redirects?

Thanks

**Comment by panva** [comment, marked as answer]
There isn't as there should not be any redirects involved. Recently the conformance certification suite was also updated to ensure no redirects are used by certified providers.

**Reply by viglucci** [reply]
Thanks for the answer. I can work around this by using the configuration in my app to provide the final URI for the OID config endpoint.
